topics/cargo-generate/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: rust-app-template scaffolding rust-app-scaffolding
```

Comment:
None of these aliases are used by any repos right now except for "scaffolding," which is [too general to be an alias](https://github.com/github/explore/blob/main/docs/API.md#aliases) here:

> You should only list an alias for your topic if the majority of repositories using that alias are referring to the same subject matter as the topic. You should not list another topic as an alias if the alias is a superset of your topic.
>
> For example, `api` makes sense in the related field for the `graphql` topic, but because many repositories tagged with `api` are not be associated with `graphql`, `api` does not make sense in the aliases field for `graphql`.

Also, they should be separated by commas if they _are_ included. But in this case I think it would be better to take them out.

Reply:
Note: the reason I have aliased the topic without the `-` is that in the docs we encourage people to share with the hashtag `#cargogenerate` on twitter (since they don't support the dash).